.jenkins/deploy/casc-configmap.yaml

```diff
@@ -94,6 +94,11 @@ data:
                     // 1 : Forks in the same account
                     // 2 : Nobody
                 }
+                // Custom Github Notification Context; https://github.com/jenkinsci/github-scm-trait-notification-context-plugin
+                traits << 'org.jenkinsci.plugins.githubScmTraitNotificationContext.NotificationContextTrait' {
+                   contextLabel("continuous-integration/jenkins/ocp-${OCP_VERSION}")
```

Comment:
It looks like this didn't work to pass through the OCP_VERSION from the env

Reply:
Did it have no effect at all, or did it pass through "continuous-integration/jenkins/ocp-" ?

Reply:
It passed through "continuous-integration/jenkins/ocp-"

Actually, I didn't have that env var set, but when I tried again, the config was invalid.

Reply:
I've tried this again today, and it seemed to work with OC 4.10

I'll test it with another version of OCP, in case it's something to do with how the env vars are parsed.
It could also be a mistake on my part.